config/initializers/graphql_cache.rb

```diff
@@ -0,0 +1,36 @@
+GraphQL::Cache.configure do |config|
+  config.namespace = 'compliance' # Cache key prefix for keys generated by graphql-cache
+  config.cache     = Rails.cache      # The cache object to use for caching
+  config.logger    = Rails.logger     # Logger to receive cache-related log messages
+  config.expiry    = 15552000         # 6 months (in seconds)
+end
+
+module GraphQL
+  module Cache
+    module DeconstructorExtensions
+      def perform
+        return raw.value if method == 'lazy'
```

Comment:
`GraphQL::Cache::Deconstructor` figures out which sort of value each field is returning. Based on that, the current methods cannot handle lazy methods, or Promises. This is just a way to handle those.